src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -6,6 +6,9 @@
         transport.send("bramble-console", data);
     }
 
+    // console namespace
+    const consoleNS = console;
```

Comment:
`const` has been supported for years by all browsers, why would we need to transpile it?